x-pack/plugins/apm/server/lib/alerts/chart_preview/get_transaction_error_rate.ts

```diff
@@ -27,17 +27,20 @@ export async function getTransactionErrorRateChartPreview({
   setup,
   alertParams,
 }: {
-  setup: Setup & SetupTimeRange;
+  setup: Setup;
   alertParams: AlertParams;
 }) {
+  const { apmEventClient } = setup;
+  const { serviceName, environment, transactionType, interval, start, end } =
+    alertParams;
+
```

Comment:
minor thing but I noticed this elsewhere also: probably unintentional on your part but it's easier for a reviewer to understand the actual changes if lines are not re-ordered like here.